src/pyuvdata/uvbeam/initializers.py

```diff
@@ -24,6 +25,7 @@ def new_uvbeam(
     model_name: str = "default",
     model_version: str = "0.0",
     feed_array: npt.NDArray[str] | None = None,
+    feed_angle: npt.NDArray[float] | None = None,
```

Comment:
mount type need to be added here as well.

Reply:
Actually, it might be ok to leave it to be picked up in kwargs, but it needs to be tested to work.

Reply:
Added test in 4bd22a9260ec0b4845b31f0bc4d84d7f61e23587 -- it does catch correctly w/ kwargs